styles/starwarsffg.css

```diff
@@ -2893,4 +2893,11 @@ img {
 .destiny-flip.dPoolLight {
   background-color: #b8383b;
   color: white;
-}
\ No newline at end of file
+}
+.starwarsffg .skillsGrid.crew .dice-pool.crew {
+  margin: 0;
+}
+.starwarsffg .skillsGrid.crew {
+  justify-content: center;
+  height: 24px;
+}
```

Comment:
You'll lose these changes in the next scss compile.

Reply:
I know, I linked it to the CSS issue.